lib/matplotlib/ticker.py

```diff
@@ -688,29 +777,38 @@ def _formatSciNotation(self, s):
 
 class LogFormatter(Formatter):
     """
-    Format values for log axis;
-
+    Format values for log axis.
     """
     def __init__(self, base=10.0, labelOnlyBase=True):
         """
-        *base* is used to locate the decade tick,
-        which will be the only one to be labeled if *labelOnlyBase*
-        is ``False``
+        `base` is used to locate the decade tick, which will be the only
+        one to be labeled if `labelOnlyBase` is ``True``.
         """
         self._base = base + 0.0
         self.labelOnlyBase = labelOnlyBase
 
     def base(self, base):
-        """change the *base* for labeling - warning: should always match the
-           base used for :class:`LogLocator`"""
+        """
+        change the `base` for labeling.
+
+        .. warning::
+           Should always match the base used for :class:`LogLocator`
+
+        """
         self._base = base
 
     def label_minor(self, labelOnlyBase):
-        'switch on/off minor ticks labeling'
+        """
+        Switch minor tick labeling on or off.
+
+        ``labelOnlyBase=True`` to turn off minor ticks.
+        """
```

Comment:
I have opened up https://github.com/matplotlib/matplotlib/pull/6264 with the code change.